### PR TITLE
add homework #3

### PR DESCRIPTION
**Pull request by DaniilKorinnyi**
Homework #3

### CODE REVIEW

File: homework3/task2.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+your_age =input("Write your age: ")
```
Comment (klymenok) [inline homework3/task2.py:1]: You shouldn't use input() here. Just create a variable with a value directly

File: homework3/task3.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+my_name = input("Your name: ")
```
Comment (klymenok) [inline homework3/task3.py:1]: You shouldn't use input() here. Just create a variable with a value directly

File: homework3/task3.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+age_in_years = input("Your age: ")
```
Comment (klymenok) [inline homework3/task3.py:2]: You shouldn't use input() here. Just create a variable with a value directly

**Review by klymenok — CHANGES_REQUESTED** [review]
Please, make changes according to comments

**Review by DaniilKorinnyi — COMMENTED** [review]
Some changes in task 2 and 3

**Review by klymenok — APPROVED** [review]
Looks good

**Pull request merged**